README.md

```diff
@@ -27,8 +27,8 @@ We're hoping that DecisionBox enables you to demonstrate that your app is making
 ### Quick Walk-Through
 
 1.	Install the DecisionBox SDK and create your first Decision site.  You'll be asked to give this site a name, and a default 'classifier' will be created for you, called "passthrough."   This is actually a passthrough to your hosted LLM provider, so you'll let your app run and collect some data using this method, before replacing this line of code with a local classifier.  The difference is that all generated responses will recorded, so you can label them, and this will reveal your baseline accuracy to improve upon.
-2.	Under your Decision Site, create a new Classifier with the appropriate outputs (choices).  You'll give this classifier a name and make this "active" rather than the default passthrough classifier. 
-3.	Replace Existing Code: Replace the code in your app which is currently using OpenAI function calls or structured outputs with the DecisionBox API call to your decision site, such that the currently active classifier will be invoked.
+2.	Under your Decision Site, create a new Classifier with the appropriate outputs (choices).  You'll give this classifier a name and make this "default" rather than the automatically-created passthrough classifier. 
+3.	Replace Existing Code: Replace the code in your app which is currently using OpenAI function calls or structured outputs with the DecisionBox API call to your decision site, such that the decision site's default classifier will be invoked.
```

Comment:
remove passive voice to make more concise

to me, using function calls implies use of structured outputs, there fore we don't need to mention both

Reply:
CC @jwyss-allday

Reply:
Follow up in https://linear.app/blueteam/issue/BT-892/revamp-readme-and-docs-to-be-consistent-with-oss-user-expectations